docs/zkapps/how-zkapps-work.mdx

```diff
@@ -24,153 +24,139 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 zkApps run in the latest versions of [Chrome](https://www.google.com/chrome/index.html), [Firefox](https://www.mozilla.org/en-US/firefox/new/), [Edge](https://microsoft.com/edge), and [Brave](https://brave.com/) web browsers.
 
-zkApps are written in TypeScript using the Mina zkApp CLI. Before you follow the guided steps in [How to write a zkApp](how-to-write-a-zkapp), get started by learning how zkApps work.
+zkApps are written in TypeScript using the Mina zkApp CLI. 
 
 A zkApp consists of two parts: 
 
-1) A smart contract
-2) A user interface (UI) for users to interact with the zkApp
+- A smart contract
+
+- A user interface (UI) for users to interact with the zkApp
 
 <figure>
-  <img src="/img/3_zkApps_Structure.jpg" width="100%" />
-  <figcaption>zkApps consists of two parts: 1) a smart contract and 2) a UI (user interface) for users to interact with it.</figcaption>
+  <img src="/img/3_zkApps_Structure.jpg" width="100%" alt="Diagram showing two parts of a zkApp" />
 </figure>
 
-<br />
-
-:::note
-
-The term <b>“smart contract”</b> refers to the code written with SnarkyJS.
-
-The term <b>“zkApp”</b> refers to the UI + the smart contract.
+- The term _smart contract_ refers to the code written with SnarkyJS.
 
-:::
+- The term _zkApp_ refers to the UI + the smart contract.
 
-### Zero-knowledge based smart contracts
+## Zero knowledge-based smart contracts
 
-Because zkApps are based on **zero-knowledge proofs (zk-SNARKs)**, a zkApp developer writes what is called a **“circuit”**. A circuit is the method from which a **prover function** and a corresponding **verifier function** are derived during the build process.
+zkApps are based on zero knowledge proofs (zk-SNARKs). As a zkApp developer, you use the zkApp CLI to scaffold and deploy your project. 
 
-The **prover function** is the function that executes a smart contract’s custom logic.
+A circuit is the method from which a prover function and a corresponding verifier function are derived during the build process.
 
-The **prover function** runs in a user’s web browser as part of the zkApp. When interacting with a zkApp's UI, users enter any data (e.g. “buy ABC for y price”) required as input to the prover function, which then generates a zero-knowledge proof.
+The prover function is the function that executes a smart contract's custom logic and runs in a user's web browser as part of the zkApp. When interacting with a zkApp UI, users enter any data (for example, buy ABC for y price) that is required as input to the prover function, which then generates a zero knowledge proof.
```

Comment:
probably worth mentioning that the prover function actually generated a proof of the executed code!